AK/Array.h

```diff
@@ -37,6 +37,13 @@ struct Array {
         return array;
     }
 
+    static constexpr Array from_fill(T const& value)
```

Comment:
I'm really not a fan of the name, here are my propositions:
 - `Array::from_value()`
 - `Array::reapeated()`

Reply:
I had some trouble with picking something appropriate, for example all the names with `value` in them (that I could think of) lack the information that the Array is filled with it (compared to just copying them over once).

`repeated()`, however, sounds reasonable to me, so I'll probably squash it in together with the next round of feedback unless somebody else suggests a name that I like even better.